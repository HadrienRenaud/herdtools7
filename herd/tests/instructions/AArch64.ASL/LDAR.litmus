AArch64 LDAR
{
0:X1=x;
int x=15;
}
  P0         ;
LDAR W0,[X1] ;

forall 0:X0=15