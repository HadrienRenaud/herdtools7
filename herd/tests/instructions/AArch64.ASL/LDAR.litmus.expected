Test LDAR Required
States 1
0:X0=15;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=15)
Observation LDAR Always 1 0
Hash=d7d4643d35c3d4f4f785e8c6946462c5

